scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -284,6 +284,10 @@
     (async/go
       (async/>! (pool->offers-chan pool-name) offers)))
 
+  (autoscaling? [_ _] false)
+
+  (autoscale! [_ _ _])
```

Comment:
We should include a warning or error log here that this function was invoked.

Reply:
Will do